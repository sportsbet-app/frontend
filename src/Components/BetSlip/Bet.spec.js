import React from 'react'
import { render } from '@testing-library/react'
//import renderer for snapshot
import renderer from 'react-test-renderer'

import BET from './Bet'

//test to check that the app will display when called.
describe('<Bet />', () => {
  it('should display', () => {
    let load = render(<BET/>)
    
  })

  //create a snapshot
  it('matches snapshot', () => {
    const tree = renderer.create(<BET />); // generates a DOM tree

    // snapshots are a JSON representation of the DOM tree
    expect(tree.toJSON()).toMatchSnapshot();
  });

  //check player count of 1
  describe('player count 1', () => {
    //check if data is displayed properly

    //check if button works

  })

  //check player count of 2
  describe('player count 2', () => {
    //check if data is displayed properly

    //check if button works

  })

  //check player count of 3
  describe('player count 3', () => {
    //check if data is displayed properly

    //check if button works
        
  })

})