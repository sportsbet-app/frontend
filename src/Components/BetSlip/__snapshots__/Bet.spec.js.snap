// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bet /> matches snapshot 1`] = `
<div>
  <h1>
     Player : 
    zero
  </h1>
  <p>
     
    testing
     : 
    3
      
    0
  </p>
  <button
    onClick={[Function]}
    style={
      Object {
        "color": "black",
      }
    }
  >
     X 
  </button>
</div>
`;
